### PR TITLE
terra

### PR DESCRIPTION
**Pull request by samuelsramko**

### CODE REVIEW

**Review by vlnevyhosteny — CHANGES_REQUESTED** [review]

**Comment by vlnevyhosteny** [comment]
Test for Terra not passing.

File: packages/blockchain/terra/src/lib/terra.sdk.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+const blockchain = Blockchain.TERRA
+
+export const TatumTerraSDK = (args: SDKArguments) => {
+  const api = BlockchainTerraService
+  const { ...abstractSdk } = abstractBlockchainSdk({ ...args, blockchain })
+
+  return {
+    ...abstractSdk,
+    api,
+    web3: terraClient(args),
```
[the remark this answers is not in the document]
Comment (samuelsramko) [reply packages/blockchain/terra/src/lib/terra.sdk.ts:19]: It's not web3, but I want to stick to the naming of other packages. But I could rename it to rpc or something like this and we can stick to it for other non-web3 chains as well

[the remark this answers is not in the document]
Comment (vlnevyhosteny) [reply packages/blockchain/terra/src/lib/terra.sdk.ts]: Both is reasonable. For example for Algo I did `algoWeb: web`.

[the remark this answers is not in the document]
Comment (samuelsramko) [reply packages/blockchain/terra/src/lib/terra.sdk.ts]: I went for client at the end of the day.

**Comment by samuelsramko** [comment]
> 

fixed.

**Review by vlnevyhosteny — APPROVED** [review]

**Pull request merged**